namespaces/collectd.yml

```diff
@@ -51,29 +66,54 @@ namespace:
           TODO
 
     - name: ps_vm
-      type: integer
+      type: group
```

Comment:
I understand that and we should open an issue for the nest plugin to set this differently. 
But until we have this fixed there I need to have the data model this way.
The plugin is already built and we plan to use it in the next version so the template should match for now.